tests/snapshots/tests__standard@large-example-2.lua.snap

```diff
@@ -1552,20 +1552,8 @@ script.Parent.Parent.Humanoid.Health = 0
 				if player[i].Character ~= nil then
 					local torso = player[i].Character:FindFirstChild("Torso")
 					if torso ~= nil then
-						torso.CFrame = CFrame.new(
-							torso.Position.x,
-							torso.Position.y,
-							torso.Position.z,
-							0,
-							0,
-							1,
-							0,
-							-1,
-							0,
-							1,
-							0,
-							0
-						) --math.random(),math.random(),math.random(),math.random(),math.random(),math.random(),math.random(),math.random(),math.random()) -- i like the people being upside down better.
+						torso.CFrame =
+							CFrame.new(torso.Position.x, torso.Position.y, torso.Position.z, 0, 0, 1, 0, -1, 0, 1, 0, 0) --math.random(),math.random(),math.random(),math.random(),math.random(),math.random(),math.random(),math.random(),math.random()) -- i like the people being upside down better.
```

Comment:
It doesn’t still break the line budget this way? It seems like when it breaks at the assignment, and that doesn’t fit within the budget, then it should fall back to the other approaches.

Reply:
Yep, this line is at exactly 120 chars, so is technically correct! (We exclude comments from the count)